### DISCUSSION TITLE
Get channel link base on subscribed channel on account logged in

### DISCUSSION THREAD

**Discussion by DinhHuy2010** [ThioJoe/YT-Spammer-Purge#387] [Ideas]
Could you add features get channel link base on subscribed channel on account logged in the program?

**Comment by TechStudent10** [comment]
Could you clarify a bit more?

**Comment by ThioJoe** [comment]
I think he means instead of having to enter a channel link, such as for the recent videos mode, it would let you choose from a list of channels you're subscribed to.

I don't think this would really be feasible considering for most people the list would be really long.

Unless he means just a feature to fetch all your subscriptions in general to make it easier to find the links. 

But I don't think it would make too much sense to add unless there's a bunch of demand for it.

**Reply by TechStudent10** [reply]
Yeah that's right considering my subscriptions..

![image](https://user-images.githubusercontent.com/76978184/149634067-8fe5eeb7-7ae8-4e80-9d56-6f3d61fd489e.png)

**Reply by DinhHuy2010** [reply]
what?

**Reply by KendallDoesCoding** [reply]
I have more then the limit that can be showed on YT of subscriptions, so I can't even check the entire list I'm subbed too lol, its above 1000 I think.

**Reply by TechStudent10** [reply]
holy that's a lot

**Reply by KendallDoesCoding** [reply]
indeed!, i've just subbed to all the channels i liked a few videos from, i heard that helps with recommendations too lol

**Reply by joaogelado** [reply]
quota would explode

**Reply by TechStudent10** [reply]
yes it would

**Reply by Firecul** [reply]
Yeah I don't think that's viable
![Screenshot_20220124_014040](https://user-images.githubusercontent.com/34950371/150708775-e1b895c5-c81b-4f11-a4c2-f3d671577caa.png)

**Reply by TechStudent10** [reply]
> Yeah I don't think that's viable 
![Screenshot_20220124_014040](https://user-images.githubusercontent.com/34950371/150708775-e1b895c5-c81b-4f11-a4c2-f3d671577caa.png)

You're gonna get 1000 subscriptions fast.

**Comment by KendallDoesCoding** [comment]
Like me, I got like 1200 but it can only display 1k at once.

**Comment by DinhHuy2010** [comment]
I mean you can select the channel not all of them or get range limit like 5 or 10 channels (depend on youtube api limit)
